modules/conversion-center/assets/js/frontend/handlers/contact-buttons.js

```diff
@@ -32,4 +34,22 @@ export default class ContactButtonsHandler extends Base {
 	onCloseButtonClick() {
 		this.elements.content.classList.remove( 'is-visible' );
 	}
+
+	initMessageBubbleTime() {
+		const messageBubbleTimeFormat = this.elements.messageBubbleTime.dataset.timeFormat;
+		const is12hFormat = '12h' === messageBubbleTimeFormat;
+		const time = new Intl.DateTimeFormat( 'default',
+			{
+				hour12: is12hFormat,
+				hour: 'numeric',
+				minute: 'numeric',
+			} ).format( new Date() );
+		this.elements.messageBubbleTime.innerHTML = time;
+	}
```

Comment:
Can you not just render it with PHP? Or do we need the visitor's time (client side)?

Reply:
Yeah, we need this to be the time on client side, not server side